packages/bot-web-ui/src/stores/load-modal-store.ts

```diff
@@ -200,77 +267,88 @@ export default class LoadModalStore {
 
         this.setActiveTabIndex(0); // Reset to first tab.
         this.setLoadedLocalFile(null);
-    }
+    };
 
     @action.bound
-    onZoomInOutClick(is_zoom_in) {
+    onZoomInOutClick = (is_zoom_in: string): void => {
         if (this.preview_workspace) {
             this.preview_workspace.zoomCenter(is_zoom_in ? 1 : -1);
         }
-    }
+    };
 
     @action.bound
-    previewRecentStrategy(workspace_id) {
+    previewRecentStrategy = (workspace_id: string): void => {
         this.setSelectedStrategyId(workspace_id);
-
         if (!this.selected_strategy) {
             return;
         }
 
-        if (!this.recent_workspace || !this.recent_workspace.rendered) {
-            const ref = document.getElementById('load-strategy__blockly-container');
+        const ref = document.getElementById('load-strategy__blockly-container');
 
-            if (!ref) {
-                // eslint-disable-next-line no-console
-                console.warn('Could not find preview workspace element.');
-                return;
-            }
-
-            this.recent_workspace = Blockly.inject(ref, {
-                media: `${__webpack_public_path__}media/`,
-                zoom: {
-                    wheel: true,
-                    startScale: config.workspaces.previewWorkspaceStartScale,
-                },
-                readOnly: true,
-                scrollbars: true,
-            });
+        if (!ref) {
+            // eslint-disable-next-line no-console
+            console.warn('Could not find preview workspace element.');
```

Comment:
when does it happen ?

Reply:
this was the legacy code whenever we dont have the` load-strategy__blockly-container` this method will fire